src/coreclr/vm/arm/stubs.cpp

```diff
@@ -1752,9 +1750,7 @@ VOID StubLinkerCPU::EmitComputedInstantiatingMethodStub(MethodDesc* pSharedMD, s
     bool isRelative = MethodTable::VTableIndir2_t::isRelative
                       && pSharedMD->IsVtableSlot();
 
-#ifndef FEATURE_NGEN_RELOCS_OPTIMIZATIONS
     _ASSERTE(!isRelative);
```

Comment:
I assume that cleaning up of these conditions and types like `RelativePointer` or `FixupPointer` is left for future change.

Reply:
Yes, that is left for a future change.